### PR TITLE
feat(prompt): add preset prompt

### PR DESCRIPTION
**Pull request by JugurthaK**
Hello, it's me again ! 

I've made a little improvement but I still don' know how to make clean PRs :(. 

Thank you,

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -90,6 +91,17 @@ async function run () {
...
+          name: 'usePreset',
+          type: context.usePreset ? null : 'select',
+          message: 'Which bundle would you like to install?',
+          initial: 1,
+          choices: [
+            { title: 'Default (Vuetify)', value: 'default' },
+            { title: 'Base (Vuetify, VueRouter)', value: 'base' },
+            { title: 'Essentials (Vuetify, VueRouter, Pinia)', value: 'essentials' },
+          ],
+        },
```
Comment (ElijahKotyluk) [inline src/index.ts:104]: This should be moved up before the typescript prompt.

File: src/index.ts [hunk 2]
```diff
@@ -123,7 +136,7 @@ async function run () {
 
   const rootTemplatePath = resolve(__dirname, '../template')
   const jsOrTs = useTypeScript || argv.typescript ? 'typescript' : 'javascript'
-  const preset = !!argv.preset ? argv.preset : 'default'
+  const preset = argv.preset || usePreset
```
Comment (ElijahKotyluk) [inline src/index.ts:139]:
```suggestion
```

File: src/index.ts [hunk 3]
```diff
@@ -123,7 +136,7 @@ async function run () {
...
 
   console.log('\n◌ Generating scaffold...')
   renderTemplate(resolve(rootTemplatePath, jsOrTs, preset), projectRoot)
```
Comment (ElijahKotyluk) [inline src/index.ts:142]:
```suggestion
  renderTemplate(resolve(rootTemplatePath, jsOrTs, usePreset), projectRoot)
```

**Review by ElijahKotyluk — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (JugurthaK) [reply src/index.ts]: It's done on my new commit

File: src/index.ts [hunk 4]
```diff
@@ -70,6 +71,17 @@ async function run () {
           },
           message: prev => `The project path: ${resolve(cwd, prev)} already exists, would you like to overwrite this directory?`,
         },
+        {
+          name: 'usePreset',
+          type: context.usePreset ? null : 'select',
+          message: 'Which bundle would you like to install?',
```
Comment (ElijahKotyluk) [inline src/index.ts:77]:
```suggestion
          message: 'Which preset would you like to install?',
```

**Review by ElijahKotyluk — CHANGES_REQUESTED** [review]

**Pull request merged**